piv/yubikey-piv-manager.md

```diff
@@ -8,4 +8,4 @@ The [Yubikey PIV Manager](https://developers.yubico.com/yubikey-piv-manager) is
 
 The command-line utility is more powerful (e.g. allows overriding pin and touch default slot policies) but it also more prone to user errors. It's very easy to overwrite an existing key, for instance.
 
-After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](#device-initialization).
+After [downloading](https://developers.yubico.com/yubikey-piv-manager) and installing Yubikey PIV Manager and `yubico-piv-tool`, insert your Yubikey. You will be prompted [initialize your device](device-initialization).
```

Comment:
I think it needs to be `../device-initialization/README.md`. Can you do that change for me?